gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -99,3 +101,25 @@ hr.top-bar{
     padding: 0;
   }
 }
+
+@media (min-width: 550px){
+  .bioimage{
+    margin-right: 2%;
+    width: 13em;
+  }
+  .flex_box{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+  }
+  .pf_field_wrapper{
+    flex-grow: 1;
+  }
+}
+
+@media (max-width: 550px){
+  .bioimage{
+    margin-right: 1.5em;
```

Comment:
We don't need any `margin-right` here, because this is single-column mode where nothing will be rendered beside the image anyway.